src/client/components/ShoppingCart/ModifyProductQuantity.js

```diff
@@ -2,24 +2,58 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import './ShoppingCart.styles.css';
 
-function ModifyProductQuantity(props) {
-  const { product, shoppingCart } = props;
-
+function ModifyProductQuantity({product,shoppingCart,setTotalPrice,setTotalDiscount,setTotalPayment}) {
   const [counter, setCounter] = useState(product.quantity);
 
   const incrementCounter = () => {
     setCounter(counter + 1);
 
     const result = shoppingCart.find(({ id }) => id === product.id);
     result.quantity += 1;
+
+    const totalPriceOfProducts = shoppingCart.reduce(
+      (sum, item) => sum + item.quantity * item.price,
```

Comment:
Example of destructuring:

```suggestion
      (sum, {price, quantity}) => sum + quantity * price,
```